clickhouse_search/models.py

```diff
@@ -581,6 +581,651 @@ class ClinvarMito(BaseClinvarJoin):
     class Meta(BaseClinvarJoin.Meta):
         db_table = 'GRCh38/MITO/reference_data/clinvar'
 
```

Comment:
I think we should transition from a single large models.py file into a multi-file models package -
its very straightforward to do (https://docs.djangoproject.com/en/4.2/topics/db/models/#organizing-models-in-a-package) and I think it will make it a lot easier to maintain if we separate out the models into 2 or 3 files (I was thinking one for reference data and one for search models but am open to suggestions). I'm okay if we put that in its own PR or just do it now